broker-core/src/main/resources/io/zeebe/broker/exporter/debug/index.html

```diff
@@ -0,0 +1,185 @@
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <div class="btn-group-toggle" data-toggle="buttons">
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-workflow-instance" type="checkbox" autocomplete="off" checked>Workfow Instance
+          </label>
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-job" type="checkbox" autocomplete="off" checked>Job
+          </label>
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-incident" type="checkbox" autocomplete="off" checked>Incident
+          </label>
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-timer" type="checkbox" autocomplete="off" checked>Timer
+          </label>
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-message" type="checkbox" autocomplete="off" checked>Message
+          </label>
+          <label class="btn btn-outline-success active" onclick="fetchRecords()">
+            <input id="filter-deployment" type="checkbox" autocomplete="off" checked>Deployment
+          </label>
+          <label class="btn btn-outline-success" onclick="fetchRecords()">
+            <input id="filter-other" type="checkbox" autocomplete="off">Other
+          </label>
+        </div>
+      </li>
+    </ul>
+    <div class="form-inline my-2 my-lg-0">
+      <form id="form-search" class="form-inline my-2 my-lg-0">
+				<div class="input-group">
+					<input id="filter-search" class="form-control my-2 my-sm-0" type="search" placeholder="Search" aria-label="Search">
+					<div class="input-group-append">
+						<button class="btn btn-outline-secondary my-2 my-sm-0" type="button" onclick="$('#filter-search').val('');fetchRecords();">Clear</button>
+						<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+					</div>
+				</div>
+      </form>
+    </div>
+  </div>
+    </nav>
+    <div class="row">
+    <div id="content" class="col"></div>
+    </div>
+  </div>
+<script id="record-template" type="x-tmpl-mustache">
+  <table class="table table-hover">
+  <thead>
+    <tr>
+      <th scope="col">Record</th>
+      <th scope="col">Key</th>
+      <th scope="col">BPMN Process Id</th>
+      <th scope="col">Element Id</th>
+      <th scope="col">Workflow Instance Key</th>
+      <th scope="col">Scope Instance Key</th>
+      <th scope="col">Element Instance Key</th>
+    </tr>
+  </thead>
+  <tbody>
+  {{ #. }}
+    <tr data-toggle="collapse" data-target="#collapse-{{_id}}" aria-expanded="false" aria-controls="collapse-{{_id}}">
```

Comment:
It should be more clear that I can click on the record to see the details. I found this feature by a fluke ;)

Reply:
Any ideas how to visualize this? Maybe a plus icon in as first column?

Reply:
Yeah, this would be an option. Or change the mouse symbol.